src/main/java/com/github/dockerjava/api/model/ResponseItem.java

```diff
@@ -100,13 +100,25 @@ public boolean isErrorIndicated() {
         private static final long serialVersionUID = -1954994695645715264L;
 
         @JsonProperty("current")
-        long current;
+        public long current;
 
         @JsonProperty("total")
-        long total;
+        public long total;
 
         @JsonProperty("start")
-        long start;
+        public long start;
+
+        public long getCurrent() {
```

Comment:
Targeted PR against 2.x branch, so leaving primitives. I will change to objects while merge into master.